charts/argo-workflows/scripts/update-image-tag.sh

```diff
@@ -20,7 +20,7 @@ if [[ $(<"${FILE}") = "${IMAGE_TAG}" ]]; then
 elif [[ "${LATEST_GIT_SHA}" = "${IMAGE_TAG}" ]] || [[ "${MANUAL_DEPLOY}" = true ]]; then
   git checkout -b "${BRANCH}"
 
-  echo "${IMAGE_TAG}" >"${FILE}"
+  tag="${IMAGE_TAG}" yq -i '.image_tag = env(tag)' "${FILE}"
```

Comment:
Might be slightly more readable without the extra variable. (I'm assuming IMAGE_TAG is already in the environment; if not then just needs an `export IMAGE_TAG` before this line.)
```suggestion
  yq -i '.image_tag = env(IMAGE_TAG)' "${FILE}"
```

Reply:
thanks, it is an env already so I will not create a new one.